scripts/deploy.sh

```diff
@@ -171,7 +171,7 @@ if [[ "${DEPLOY_OA}" == "yes" ]]; then
 
   if [[ "${DEPLOY_TEMPEST}" == "yes" ]]; then
     # Deploy tempest
-    run_ansible os-tempest-install.yml
+    run_ansible ${BASE_DIR}/scripts/run_tempest.yml
```

Comment:
Yes, the AIO bootstrap overrides it by default so you'd need to make sure that the RPCO AIO bootstrap replaces that override just like you're doing with ``tempest_run``.